code/__DEFINES/surgery_tools.dm

```diff
@@ -0,0 +1,103 @@
+// Order of the tools matters. The most specific path should go first. 
+// /obj/item/stack/medical/bruise_pack/advanced should go before /obj/item/stack/medical/bruise_pack
+
+#define SURGERY_TOOLS_INCISION list(/obj/item/scalpel = 100,		\
+									/obj/item/kitchen/knife = 90,	\
+									/obj/item/shard = 60, 		\
+									/obj/item/scissors = 12,		\
+									/obj/item/twohanded/chainsaw = 1, \
+									/obj/item/claymore = 6, \
+									/obj/item/melee/energy/ = 6, \
+									/obj/item/pen/edagger = 6)
+
+#define SURGERY_TOOLS_CLAMP 	list(/obj/item/scalpel/laser = 100, \
+									/obj/item/hemostat = 100,	\
+									/obj/item/stack/cable_coil = 90, 	\
+									/obj/item/assembly/mousetrap = 25)
+
+#define SURGERY_TOOLS_RETRACT_SKIN 	list(/obj/item/scalpel/laser/manager = 100, \
+										/obj/item/retractor = 100, 	\
+										/obj/item/crowbar = 90,	\
+										/obj/item/kitchen/utensil/fork = 60)
+
+#define SURGERY_TOOLS_CAUTERIZE		list(/obj/item/scalpel/laser = 100, \
+										/obj/item/cautery = 100,			\
+										/obj/item/clothing/mask/cigarette = 90,	\
+										/obj/item/lighter = 60,			\
+										/obj/item/weldingtool = 30)
```

Comment:
There doesn't seem to be any checks done here to ensure the tool is actually working (e.g. a welder will always act as a cautery, regardless of if it's lit). It might be an idea in `/datum/surgery_step/proc/initiate` to include a `if(!item.tool_enabled)` to fail if the tool isn't enabled

It'll probably need a bit of work for cigarettes and the like to not work as surgical tools if unlit, but that can come at a later date and this tweak shouldn't impact on any surgical behaviours at present

Reply:
Placed it in the initiate.
A later PR can use tool_enabled for cigs etc as well. Instead of the lit var they use there. Out of the scope of this one though.
Thanks for the remark! Fixes another case as well so nice